src/routes/+layout.svelte

```diff
@@ -105,5 +107,27 @@
 	</svelte:fragment>
 	<!-- Page Route Content -->
 	<slot />
-	<svelte:fragment slot="footer">Page Footer</svelte:fragment>
+	<svelte:fragment slot="pageFooter">
+			<div class="grid grid-cols-2 place-items-center md:!grid-cols-4 gap-1 lg:ml-10 lg:mr-10">
+
+				<enhanced:img
+					src={FooterImg1}
+					alt="Ulrich Becher in Basel"
+				/>
+
+				<enhanced:img
+					src={FooterImg2}
+					alt="Ulrich Becher in Basel"
+				/>
+
+				<p class="h5 md:h6 lg:h5 text-black">
+					Ein Prototyp des Deutschen Exilarchivs (DEA) und des Schweizerischen Literarurarchivs (SLA) 
+				</p>
+
+				<p class="h5 text-black">
```

Comment:
```suggestion
				<p class="h5 text-black col-span-2 md:col-span-1 justify-self-start md:justify-self-end">
```

Reply:
ohh now i see. I miss understood. I thought you meant span as in the verb